pkgs/applications/graphics/fig2dev/default.nix

```
@@ -1,26 +1,33 @@
-{ lib, stdenv, fetchurl, ghostscript, libpng } :
+{ lib, stdenv, fetchurl, ghostscript, libpng, makeWrapper
+, coreutils, bc, gnugrep, gawk, gnused } :
 
 let
-  version = "3.2.7b";
-
+  version = "3.2.8";
 in stdenv.mkDerivation {
   pname = "fig2dev";
   inherit version;
```

```suggestion
in stdenv.mkDerivation rc {
  pname = "fig2dev";
  version = "3.2.8";
```

that let in is totally not required.